pandas/sparse/series.py

```diff
@@ -570,8 +570,7 @@ def copy(self, deep=True):
         return self._constructor(new_data, sparse_index=self.sp_index,
                                  fill_value=self.fill_value).__finalize__(self)
 
-    def reindex(self, index=None, method=None, copy=True, limit=None,
```

Comment:
leave the signature as it was
use a shared_doc to construct the doc-strings (see how its done in Series itself).